client/packages/common/src/intl/locales/en/common.json

```diff
@@ -149,6 +149,8 @@
   "label.notes": "Notes",
   "label.num-packs": "# Packs",
   "label.number": "Number",
+  "label.auth-status": "Authorisation Status",
```

Comment:
:nit this would look better as `Authorisation` so that all column headers are single line. I'd add a detail to show the additional text

Reply:
Can this possible be done in front end changes PR, as a follow up ?